roles/analysis/vars/main.yml

```diff
@@ -1,2 +1,6 @@
 ---
 # vars file for analysis
+convert2rhel_enable_repos_args: "{{ ('--enablerepo ' + convert2rhel_repos_enabled | default([], true) | join(' --enablerepo ')) if convert2rhel_repos_enabled | length > 0 else '' }}"
+
+result_filename_prefix: /var/log/convert2rhel/convert2rhel-pre-conversion
```

Comment:
Are we missing the extension here? I think we need `.json`, right?

Reply:
I don't think so, we can discuss what is best approach, but I did similar thing as leapp have to have only the common path in variable (both in analysis and parse role). In analysis the variable is used only in message outputed to user (And .txt is added after it), in parse there is variable to json report.

Reply:
I did make it consistent in both roles, analysis role has only `result_filename_txt` (used in output message), but parse_c2r_report has both `result_filename_txt` and `result_filename_json`